incubator/nodejs-express/stack.yaml

```diff
@@ -11,3 +11,4 @@ default-template: simple
 requirements:
   docker-version: ">= 17.09.0"
   appsody-version: ">= 0.2.7"
+ 
```

Comment:
This can be trimmed out.

Reply:
Yes, will remove.
Regarding incrementing the stack version, we haven't release this version of the stack yet, so there should not be any reason to increment.